azure-pipelines.yml

```diff
@@ -174,6 +174,46 @@ jobs:
           trigger_sha: '$(trigger_sha)'
       - template: ci/report-end.yml
 
+  - job: compatibility
+    dependsOn:
+      - check_for_release
+    variables:
+      release_sha: $[ dependencies.check_for_release.outputs['out.release_sha'] ]
+      release_tag: $[ coalesce(dependencies.check_for_release.outputs['out.release_tag'], '0.0.0') ]
+      trigger_sha: $[ dependencies.check_for_release.outputs['out.trigger_sha'] ]
+      is_release: $[ dependencies.check_for_release.outputs['out.is_release'] ]
+    timeoutInMinutes: 60
+    pool:
+      name: 'linux-pool'
+    steps:
+      - template: ci/report-start.yml
+      - checkout: self
+      - bash: |
+          set -euo pipefail
+          eval "$(./dev-env/bin/dade-assist)"
+          git checkout $(release_sha)
+          git checkout $(trigger_sha) -- docs/source/support/release-notes.rst
+        name: checkout_release
+        condition: eq(variables.is_release, 'true')
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
```

Comment:
Is the Bazel configuration done in the root WORKSPACE going to translate to the nested one?

Reply:
Good catch! I’ve added an environment variable and modified the script to write to the bazelrc.local in that directory. Hopefully that should work.